### PR TITLE
Add latency graph command

### PR DESCRIPTION
**Pull request by K4ST0R**
### Description

Issue #1954 

I have added the implementation of the following command to the project [LATENCY GRAPH](https://redis.io/commands/latency-graph).

I needed to change docker's configuration to be able to test the command.
---

### Checklist

- [x] Does `npm test` pass with this change (including linting)?
- [x] Is the new or changed code fully tested?
- [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?

### CODE REVIEW

**Comment by leibale** [comment]
@K4ST0R Thanks for contributing! Wanna review my changes? :pray:

File: packages/client/lib/commands/LATENCY_GRAPH.spec.ts [hunk 1]
```diff
@@ -1,28 +1,30 @@
...
-      assert.equal(typeof (await client.latencyGraph('command')), 'string');
-    },
-    GLOBAL.SERVERS.OPEN
-  );
+        assert.equal(
+            typeof await client.latencyGraph('command'),
+            'string'
+        );
+    }, {
+        serverArguments: ['--enable-debug-command', 'yes']
```
Comment (K4ST0R) [inline packages/client/lib/commands/LATENCY_GRAPH.spec.ts:28]: Oh thank you for this! 😁

**Review by K4ST0R — COMMENTED** [review]
Looks good to me, thank you for reviewing my code :)

Sorry for the coding style 😅

**Comment by leibale** [comment]
No worries, code style is a matter of taste, there is no right and wrong..

File: packages/client/lib/commands/LATENCY_GRAPH.spec.ts [hunk 2]
```diff
@@ -14,17 +14,19 @@ describe('LATENCY GRAPH', () => {
         );
     });
 
-    testUtils.testWithClient('client.latencyGraph', async (client) => {
+    testUtils.testWithClient('client.latencyGraph', async client => {
         await Promise.all([
             client.configSet('latency-monitor-threshold', '1'),
-            client.sendCommand(['DEBUG', 'SLEEP', '.1'])
```
Comment (K4ST0R) [inline packages/client/lib/commands/LATENCY_GRAPH.spec.ts]: `.1`was ok, `DEBUG SLEEP`is in seconds and `latency-monitor-threshold`in ms :)

**Pull request merged**